mechanical/cad/cad-workflow.md

```diff
@@ -0,0 +1,23 @@
+## Onshape Workflow Standards
+
+This section will outline stardard workflows in Onshape for Team 1675. In general, our CAD designs will follow a top-down workflow in which most parts are derived off of a master sketch or master model. This allows critical dimensions to be defined and updated in a singular location and updates will propogate through parts and assemblies.
+
+---
+
+### Top-Down Design Training
+[Onshape - Top-Down Design](https://learn.onshape.com/learn/learning-path/top-down-design)
+
+ These training modules will show the basics of the top-down design approach that the team takes before getting into the specific ways we use these techniques in our parts and assemblies.
+
+ In these modules you will learn:
+ - Multi-part Part Studios
+ - Managed In-Context Design
+ - Designing with Master Models
+
+ *Recommended for: Robot Design Team; 3 Lessons 6-10 hours*
+
+ ---
+
+ ### 1675-Specific CAD Workflow and Standards

```

Comment:
for some reason this didn't format into a header? not sure why. if it is still doing this when you flesh out the page we will figure it ouit